congress/bills.py

```diff
@@ -0,0 +1,54 @@
+from .base import Client, SingleListParser, BaseValidator, BaseParser
+
+class Bills:
+    def __init__(self, api_key):
+        self._api_key = api_key
+
+    def search_bills(self, query):
+        client = Client(self._api_key, 
+                        SingleListParser("bills"),
+                        BaseValidator())
+        return client.get('bills/search.json?query={}'.format(query))        
```

Comment:
the get method can take keyword args and it passes these into the request.  I would prefer to allow requests to build query strings instead of doing it manually